packages/example-getting-started/docs/8-putting-it-together.md

```diff
@@ -1,61 +1,73 @@
 ### Putting it all together
 
-Now that we've got all of our artifacts made, let's set them up in our
-application!
+We've got all of our artifacts now, and all that's left is to bind them to our
+[Application](http://loopback.io/doc/en/lb4/Application.html) so that LoopBack's
+[Dependency injection](http://loopback.io/doc/en/lb4/Dependency-injection.html)
+system can tie it all together for us!
 
-We'll define a new helper function for setting up the repositories, as well
-as adding in our new controller binding.
+We'll define a new helper function for setting up the repositories, and
+LoopBack's [boot module](https://github.com/strongloop/loopback-next/tree/master/packages/boot) will
+take care of the other artifacts for us!
```

Comment:
How will it take care of the other artifacts? Why do these artifacts even need to be taken care of in the first place? By what method? What is a `binding`?